src/components/dashboard/purchase-card.svelte

```diff
@@ -30,7 +30,7 @@
       <time>{new Date(purchase.time).toLocaleString()}</time>
       <div class="purchaser">
         {(-purchase.amount)} item{purchase.amount < -1 && 's' || ''} purchased by {purchase.account.full_name}
-        <span on:click={copy} class="copy-email">click to copy e-mail</span>
+        <a on:click={copy} href="# " class="copy-email">click to copy e-mail</a>
```

Comment:
Yeah, it's not really conventional to use # in Sapper routes, mainly for the reason that this link will now bring the user to `/# ` as opposed to `/dashboard# `. I guess we could use a span, just give it the styles of a link:
```scss
text-decoration: underline;
color: $main;
cursor: pointer;
```